### ISSUE TITLE
Asset pipeline

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#687]
**TL;DR**: An asset pipeline in place to support javascript changes. 

**Narrative**
An asset pipeline would give us:
- Faster view update while making changes
- Faster page loads
- Treeshaking
- More intuitive Javascript code
- Sass and the like

**Acceptance Criteria**
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] Tests
- [ ] Scoping
- [ ] Alternatives Considered
- [ ] Rico sign-off

**Implementation Notes**
Things I found while looking for solutions to a Django asset pipeline:
- https://django-pipeline.readthedocs.io/en/latest/
- https://pypi.org/project/django-webpack-loader/

Additional Reading that may or may not support my point:
- https://guides.rubyonrails.org/asset_pipeline.html
- https://medium.com/@tomkadwill/should-you-use-the-asset-pipeline-78ebb943e589
